doc/meetings/2016-07-13.md

```diff
@@ -0,0 +1,117 @@
+# Attendees
+
+* Myles Borins
+* Michael Dawson
+* Jeremiah Senkpiel
+* Ali Sheikh
+* Michael Hablich
+* Franziska Hinkelmann
+* Rod Vagg
+
+# Agenda
+
+* Managing v8 floating patches
+* v4.5
+* Exporting ssl symbols on windows
+
+# Discussion
+
+## Managing v8 floating patches - https://github.com/nodejs/LTS/issues/111
+
+* V8 team willing to have place where floating patching can land
+* Overlap of 2-3 weeks were still maintained that might cause conflicts
+* would need to create separate branch for node.
+* Why not create fork
+* Notes from Ben N
+  * What timeframe for using google infrastructure
+  * We have our own infra and can run from Node tree
+
+### Infrastructure
+* Chromium infrastructure team is moving fast and deprecating lots of things so may
+  not support v8 version older than 1 year.
+* Will need special (LTS like) support for Node.
+* Jeremiah, backport changes need to allow older tests to run on new infra.
+* Can still run locally, but cannot run old way in google infra.
+* Michael (google) not a feasible option to run Node.js older branches
+  after a few months.
+* Google may be able to assist in adding resources to the Node CI.
```

Comment:
I don't think any specifics were named AFAIK. Instead of leveraging Chromium's infra it might make more sense to beef up Node's instead.

I have not looked into the specifics though. Probably makes sense to reword it:

Is it possible that Google provides resources for Node's CI?